test/components/frameTest.js

```diff
@@ -150,6 +150,44 @@ describe('clone tab', function () {
     })
   })
 
+  describe('index', function () {
+    Brave.beforeAll(this)
+
+    before(function * () {
+      this.clickWithTargetPage = Brave.server.url('click_with_target.html')
+      this.page1 = Brave.server.url('page1.html')
+
+      yield setup(this.app.client)
+      yield this.app.client
+        .waitUntilWindowLoaded()
+        .windowByUrl(Brave.browserWindowUrl)
+        .waitForUrl(Brave.newTabUrl)
+        .url(this.clickWithTargetPage)
+        .waitForVisible('#name')
+        .click('#name')
+      yield this.app.client
+        .windowByUrl(Brave.browserWindowUrl)
+        .waitForExist('.tab[data-frame-key="2"]')
+      yield this.app.client
+        .ipcSend('shortcut-set-active-frame-by-index', 0)
+        .windowByUrl(Brave.browserWindowUrl)
+        .ipcSend(messages.SHORTCUT_ACTIVE_FRAME_CLONE)
+        .windowByUrl(Brave.browserWindowUrl)
+        .waitForExist('.tab[data-frame-key="3"]')
+    })
+
+    it('inserts after the tab to clone', function * () {
+      var clickWithTargetPage = this.clickWithTargetPage
+      yield this.app.client
+        .windowByUrl(Brave.browserWindowUrl)
+        .ipcSend('shortcut-set-active-frame-by-index', 1)
```

Comment:
I think it might be better in this case to use webview:nth-child to check the webview src. Might even be useful to check all 3 of them and make sure the framekey => nth-child index is what you expect for all three